pkg/api/graph/graph.go

```diff
@@ -158,7 +158,7 @@ func (g Graph) SyntheticNodes() []graph.Node {
 	sort.Sort(SortedNodeList(nodeList))
 	for _, node := range nodeList {
 		if potentiallySyntheticNode, ok := node.(ExistenceChecker); ok {
-			if potentiallySyntheticNode.Found() {
+			if !potentiallySyntheticNode.Found() {
```

Comment:
Yeah, this is embarrassing.  I'm rolling it into https://github.com/openshift/origin/pull/3420